apps/web/templates/about.html

```diff
@@ -0,0 +1,213 @@
+      <ul>
+        <li><a href="#">Sobre</a></li>
+        <li><a href="#">Eventos</a></li>
+        <li><a href="#">Notícias</a></li>
+        <li><a href="#">Produtos</a></li>
+      </ul>
+    </li>
+    <li><a href="#">Publicações</a></li>
+    <li><a href="#">Galeria</a></li>
+    <li><a href="#">Regionais</a></li>
+    <li><a href="#">Contato</a></li>
+  </ul>
+</nav><!-- #nav-menu-container -->
+{% endblock menu %}
+
+{% block intro %}
+<!--==========================
+    Intro Section
+============================-->
+<section id="intro">
+  <div class="intro-content">
+    <h2>SOCIEDADE BRASILEIRA DE GEOLOGIA – NÚCLEO RIO DE JANEIRO</h2>
+    <div class="container">
+      <img src="../static/img/geology-icon.png"></img>
+    </div>
+    <br>
+    <h5 class="about-quote">
+      "Temos como o principal objetivo fomentar o desenvolvimento e divulgação da
+      geociências na sociedade."
+    </h5>
+  </div>
+  <div id="intro-carousel" class="owl-carousel">
+    <div class="item" style="background-image: url({% static 'img/intro-carousel/1.jpg' %});"></div>
+    <div class="item" style="background-image: url({% static 'img/intro-carousel/2.jpg' %});"></div>
+    <div class="item" style="background-image: url({% static 'img/intro-carousel/3.jpg' %});"></div>
+    <div class="item" style="background-image: url({% static 'img/intro-carousel/4.jpg' %});"></div>
+    <div class="item" style="background-image: url({% static 'img/intro-carousel/5.jpg' %});"></div>
+  </div>
+</section><!-- #intro -->
+{% endblock  %}
+
+{% block content %}
+<!--==========================
+      About Section
+    ============================-->
+<section id="about" class="wow fadeInUp">
+  <div class="container">
+    <div class="section-header">
+      <h2>A SBG-RJ</h2>
+    </div>
+    <div class="row">
+      <div class="col-lg-6 about-img">
+        <p>
+          A SBG é uma entidade técnico-científica, sem fins lucrativos, fundada em 1946. Tem
+          sede na Rua do Lago, 562, Cidade Universitária USP, São Paulo, SP, e núcleos
+          regionais com sedes em Belo Horizonte, Rio de Janeiro, Goiânia, Belém, Recife,
+          Salvador, Brasília, Curitiba, Porto Alegre e São Paulo.
+          Os núcleos regionais realizam, individualmente ou em parceria com outros núcleos, nos anos ímpares,
+          os simpósios de geologia.
+        </p>
+        <p>
+          O Núcleo Rio de Janeiro foi fundado em 1XX. A cada dois anos, nos anos pares, a
+          SBG realiza o Congresso Brasileiro de Geologia. Cabe a um dos núcleos realizá-
+          lo, por votação dos associados e com o apoio da Sede.
+        </p>
+        <p>
+          Os núcleos regionais realizam, individualmente ou em parceria com outros núcleos, nos anos ímpares,
+          os simpósios de geologia.
+        </p>
+
+        <div class="container">
+          <img src="../static/img/Rio_de_Janeiro_Mountains.jpg" width="100%%"></img>
```

Comment:
Eu renomearia essa imagem para algo mais simples, fica mais fácil de entender oq é ela.

```suggestion
          <img src="../static/img/montanhas-rio-de-janeiro.jpg" width="100%" alt="meu alt" title=" meu title">
```

**OBS:** Lembre-se do `alt` e do `title`.